packages/client/src/components/GameScore.tsx

```diff
@@ -1,28 +1,63 @@
-import { Box, Flex, Heading, Text } from 'theme-ui';
-import { GameState, Score } from 'shared';
+import { Box, Button, Flex, Heading, Text } from 'theme-ui';
+import { GameState, GameEvent, Score } from 'shared';
+import { gameIO } from 'lib/socket';
+import { useParams } from 'react-router-dom';
 
 type Props = {
   gameScore: Score[];
   gameState: GameState;
 };
 
+const END_OF_GAME_SCORE = 11;
+
 export const GameScore = ({ gameScore, gameState }: Props) => {
+  const { players } = gameState;
+  const totals = Object.fromEntries(gameScore.map((g, i) => [players[i].username, g.total]));
+
+  const isGameFinished =
+    totals[players[0].username] !== totals[players[1].username]
```

Comment:
Is the game finished if they have the same total score...? Also, do you prefer doing the `!==` check rather than `===`?
Also this can be done in this way, right?
```
const isGameFinished = totals[players[0].username] !== totals[players[1].username] && ...
```

Reply:
yes it was late and I got lost in ternaries we can do it with &&. The game is not finished if the players have the same total score.
Games end when one of the player goes over (or equal) to 11 but if both players reach the same total number than I believe you need to keep play